HoloRepositoryUI/client/src/components/app/DeviceConnectorPage.tsx

```diff
@@ -13,10 +14,10 @@ class DeviceConnectorPage extends Component<RouteComponentProps> {
           Use the code on this page to link your HoloLens application with your SMART on FHIR login.`
         ]}
       >
-        <div>This feature is not available yet.</div>
+        <div>Your authentication PIN: <b>{this.props.context!.pin}</b></div>
       </ContentContainer>
     );
   }
 }
 
-export default DeviceConnectorPage;
+export default withAppContext(DeviceConnectorPage);
```

Comment:
No idea :joy:

Reply:
Oh I remember. If you do a "shallow render" and you have a HOC (Higher Order Component), it will just mount the layer of this wrapper component. The actual component, being a child of that wrapper, will not get mounted, so it's not really a test. Therefore it would be better if you could add that to the test plase :)